etna/transforms/math/lags.py

```diff
@@ -96,3 +100,205 @@ def _transform(self, df: pd.DataFrame) -> pd.DataFrame:
     def get_regressors_info(self) -> List[str]:
         """Return the list with regressors created by the transform."""
         return [self._get_column_name(lag) for lag in self.lags]
+
+
+class ExogShiftTransform(IrreversibleTransform, FutureMixin):
```

Comment:
About the naming of the class. Are you sure that it should be named `ExogShiftTransform`? We have `LagTransform` and now we name this thing a shift. May be it can look inconsistent.

Reply:
These classes have similar implementation logic but different use cases and capabilities. This name highlights differences while preserving the meaning of transform.